bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -1327,6 +1367,27 @@ public void onNewMessageEvent(EventObject event) {
             default:
                 break;
         }
+        int preset = 0;
+        if (key.startsWith(KEY_FM_PRESET)) {
+            try {
+                preset = Integer.parseInt(key.substring(KEY_FM_PRESET.length()));
+            } catch (NumberFormatException e) {
```

Comment:
Consider adding a comment here why this exception is ignored.

Reply:
Done